core/block/import/objectid/treeobject.go

```diff
@@ -0,0 +1,40 @@
+package objectid
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/anyproto/any-sync/commonspace/object/tree/treestorage"
+
+	"github.com/anyproto/anytype-heart/core/block/import/converter"
+	"github.com/anyproto/anytype-heart/core/block/object/objectcache"
+	"github.com/anyproto/anytype-heart/core/block/object/payloadcreator"
+)
+
+type TreeObject struct {
```

Comment:
maybe unexport all unnecessary types and functions?

Reply:
Done